sdk/python/kubeflow/trainer/hf_dockerfile

```diff
@@ -0,0 +1,15 @@
+# Use an official Pytorch runtime as a parent image
+FROM pytorch/pytorch:2.0.1-cuda11.7-cudnn8-runtime
```

Comment:
I think this image wouldn't work well at all platforms (linux/amd64, linux/arm64, and `linux/ppc64le`) since this base image supports only linux/amd64.

https://hub.docker.com/layers/pytorch/pytorch/2.0.1-cuda11.7-cudnn8-runtime/images/sha256-82e0d379a5dedd6303c89eda57bcc434c40be11f249ddfadfd5673b84351e806?context=explore

So, can we use the NGC PyTorch image as a base one?

https://catalog.ngc.nvidia.com/orgs/nvidia/containers/pytorch/tags

Also, can we use the latest CUDA/PyToch version?

Reply:
done